python/examples/pylammps/README.md

```diff
@@ -0,0 +1,89 @@
+# PyLammps and Jupyter Notebooks
+
+This folder contains examples showcasing the usage of the PyLammps Python
+interface and Jupyter notebooks. To use this you will need LAMMPS compiled as
+a shared library and the LAMMPS Python package installed.
+
+An extensive guide on how to achieve this is documented in the [LAMMPS manual](https://lammps.sandia.gov/doc/Python_install.html). There is also a [PyLammps tutorial](https://lammps.sandia.gov/doc/Howto_pylammps.html).
+
+The following will show one way of creating a Python virtual environment
+which has both LAMMPS and its Python package installed:
+
+1. Clone the LAMMPS source code
+
+   ```shell
+   $ git clone https://github.com/lammps/lammps.git
```

Comment:
@rbberger @sjplimp I am wondering whether we should start recommending to clone the "unstable" or "stable" branch in these examples (and in the manual) and not the development branch (aka "master")?
`git clone -b stable https://github.com/lammps/lammps.git`

Reply:
fine by me

Reply:
ok